Geometry/TrackerGeometryBuilder/interface/pixelTopology.h

```diff
@@ -1,11 +1,52 @@
-#ifndef Geometry_TrackerGeometryBuilder_phase1PixelTopology_h
-#define Geometry_TrackerGeometryBuilder_phase1PixelTopology_h
+#ifndef Geometry_TrackerGeometryBuilder_pixelTopology_h
+#define Geometry_TrackerGeometryBuilder_pixelTopology_h
 
 #include <cstdint>
 #include <array>
 
+namespace pixelTopology {
+  template <class Function, std::size_t... Indices>
+  constexpr auto map_to_array_helper(Function f, std::index_sequence<Indices...>)
+      -> std::array<typename std::result_of<Function(std::size_t)>::type, sizeof...(Indices)> {
+    return {{f(Indices)...}};
+  }
+
+  template <int N, class Function>
+  constexpr auto map_to_array(Function f) -> std::array<typename std::result_of<Function(std::size_t)>::type, N> {
+    return map_to_array_helper(f, std::make_index_sequence<N>{});
+  }
+
+  static constexpr auto maxNumberOfLadders = 160;
```

Comment:
Why is `static` used here? It is not used in any other global `constexpr` definition in this file.

Reply:
Indeed, this shouldn't be here